docs/en_US/getting-started/getting-started.md

```diff
@@ -1,113 +1,214 @@
+
+```bash
+curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+```
+
+2. Add Helm chart repository
+
+```bash
+helm repo add emqx https://repos.emqx.io/charts
+helm repo update
+```
 
 ### Install EMQX Operator
 
-1. Install by helm
+```bash
+helm install emqx-operator emqx/emqx-operator --namespace emqx-operator-system --create-namespace
+```
+
+Check whether EMQX Operator is ready
+
+```bash
+kubectl get pods -l "control-plane=controller-manager" -n emqx-operator-system
+```
 
- ```shell
- helm repo add emqx https://repos.emqx.io/charts
- helm repo update
- helm install emqx-operator emqx/emqx-operator --namespace emqx-operator-system --create-namespace
- ```
-2. Wait EMQX Operator Controller running
+The output is similar to:
 
-```shell
-$ kubectl get pods -l "control-plane=controller-manager" -n emqx-operator-system
+```bash
 NAME                                                READY   STATUS    RESTARTS   AGE
 emqx-operator-controller-manager-68b866c8bf-kd4g6   1/1     Running   0          15s
 ```
 
-## Deploy the EMQX
+### Upgrade EMQX Operator
+
+Execute the following command to upgrade EMQX Operator, if you want to specify the upgraded version, you only need to add --version=x.x.x
+
+```bash
+helm upgrade emqx-operator emqx/emqx-operator -n emqx-operator-system
+```
+
+**Remarks:** Does not support version 1.x.x EMQX Operator upgrade to version 2.x.x
+
+### Uninstall EMQX Operator
+
+Execute the following command to uninstall EMQX Operator
+
+```bash
+helm uninstall emqx-operator -n emqx-operator-system
+```
+
+## Deploy EMQX
 
 ### Deploy EMQX 5
 
-1. Deploy EMQX Custom Resource
+1. Deploy EMQX
+
+```bash
+cat << "EOF" | kubectl apply -f -
+apiVersion: apps.emqx.io/v2alpha1
+kind: EMQX
+metadata:
+   name: emqx
+spec:
+   image: emqx/emqx:5.0.14
+EOF
+```
+
+For a complete example, please see [emqx-full.yaml](https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v2alpha1/emqx-full.yaml)
 
-    ```bash
-    cat << "EOF" | kubectl apply -f -
-      apiVersion: apps.emqx.io/v2alpha1
-      kind: EMQX
-      metadata:
-        name: emqx
-      spec:
-        image: emqx/emqx:5.0.9
-    EOF
-    ```
+For detailed explanation of each field, please refer to [v2alpha1-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v2alpha1-reference.md)
 
-    Full example please check [`emqx-full.yaml`](https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v2alpha1/emqx-full.yaml).
+2. Check whether the EMQX cluster is ready
 
-    Detailed explanation of each field please check [v2alpha1-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v2alpha1-reference.md)
+```bash
+kubectl get emqx emqx -o json | jq ".status.conditions"
+```
 
-2. Check EMQX Custom Resource status
+The output is similar to:
```

Comment:
Make sure the item has `type ：ClusterRunning` and `status：True`

Reply:
> Make sure the item has `type ：ClusterRunning` and `status：True`

OK，I know you mean